website/src/layout/components/OpenElsewhereLinks/OpenInGithub/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { useMemo } from "react"
```

Comment:
small nit but we could turn open in X a component in itself. Basically accepts a url generator and icon. But i think we're only ever gonna do github and colab really so it's probably fine.